src/runtime/storage/store-interface.ts

```diff
@@ -188,9 +182,9 @@ export abstract class ActiveMuxer<T extends CRDTTypeRecord> implements StorageCo
 
   abstract getLocalModel(muxId: string, id: number): CRDTModel<T>;
```

Comment:
these five methods can be removed from here, they are already in ActiveStore

Reply:
Done.